tools/deployment-cli-tools/ch_cli_tools/migration.py

```diff
@@ -0,0 +1,101 @@
+"""
+Utilities to perform a migration of the deployment to the latest supported version.
+"""
+import os
+import json
+
+from . import HERE
+from cloudharness_utils.constants import APPS_PATH
+from .utils import get_sub_paths, search_word_in_file, search_word_in_folder
+
+TO_CHECK = ["deploy", "tasks", "Dockerfile"]
+
+
+def perform_migration(base_root, accept_all=False):
+    all_files_detected = []
+    for file in os.listdir(os.path.join(HERE, "migrations")):
+        f = open(os.path.join(HERE, "migrations", file), "r")
+        migration_json = json.load(f)
```

Comment:
Suggestion: Once loaded, it's not a json anymore, it's a dict structure. One better name could be migration_spec